### PR TITLE
remove hard coded offsets (leftover from EDGE integration)

### PR DESCRIPTION
**Pull request by hpcowley**
When porting EDGE capabilities into pytri, accidentally included some offsets that were used in stand-alone web application. This should fix positioning issues!

### CODE REVIEW

**Comment by j6k4m8** [comment]
Thank you!

**Comment by hpcowley** [comment]
Scrapped all rescaling and repositioning. Take another look?

File: pytri/js/LargeGraphLayer.js [hunk 1]
```diff
@@ -107,12 +73,8 @@ class GraphLayer extends Layer {
             let sn = this.graph.nodes[link.source];
             let tn = this.graph.nodes[link.target];
 
-            let snPos = this._scaledPos(
-                this._getNodePosition(sn)
-            );
-            let tnPos = this._scaledPos(
-                this._getNodePosition(tn)
-            );
+            let snPos =this._getNodePosition(sn)
```
Comment (jtpdowns) [inline pytri/js/LargeGraphLayer.js]: use a space on either side of the equals, plz

File: pytri/js/ColorGraphLayer.js [hunk 2]
```diff
@@ -60,10 +60,6 @@ class ColorGraphLayer extends Layer {
         let xPos = nodeData.x - this.xSubtract
```
Comment (jtpdowns) [inline pytri/js/ColorGraphLayer.js]: Can we get rid of the subtracts here as well?

Comment (hpcowley) [reply pytri/js/ColorGraphLayer.js]: Blerg. `LargeGraphLayer` != `ColorGraphLayer`. I was editing the wrong file... thanks for your patience!

**Comment by hpcowley** [comment]
@jtpdowns , in light of new changes pushed, do you have any remaining concerns?

**Pull request merged**